docs/pipeline-configuration-files/specifications.mdx

```diff
@@ -43,20 +43,22 @@ pipelines:                      # A list of pipeline configurations
         name: my-file-destination
         settings:
           path: ./file2.txt
-        processors:             # A list of processor configurations, processors are attached to the connector
-          - id: proc1           # Processor ID [required]
-            type: js            # Processor type [required]
-            workers: 2          # Number of parallel workers
-            settings:           # A map of configuration keys and values for the processor (specific to the chosen type)
+        processors:                   # A list of processor configurations, processors are attached to the connector
+          - id: proc1                 # Processor ID [required]
+            plugin: custom.javascript # Processor plugin name [required]
+            condition: "{{ eq .Metadata.foo "bar" }}"  # Condition (Go template expression) that dictates if the record will be passed to the processor or not.
```

Comment:
One last thing, since this wouldn't be a valid yaml otherwise:
```suggestion
            condition: '{{ eq .Metadata.foo "bar" }}'  # Condition (Go template expression) that dictates if the record will be passed to the processor or not.
```